tests/util/kube_utils.go

```diff
@@ -757,64 +753,24 @@ func CheckPodRunning(n, name string, kubeconfig string) error {
 // CreateMultiClusterSecrets will create the secrets and configmap associated with the remote cluster
 func CreateMultiClusterSecrets(namespace string, KubeClient kubernetes.Interface, RemoteKubeConfig string, localKubeConfig string) error {
```

Comment:
One nit.   I don't think you need the kubernetes.Interface passed anymore.  Will check on my system in a bit.

Reply:
Yeah, remove KubeClient